### PR TITLE
Move server options with other server stuff

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

File: ui/menu_ingame.rml [hunk 1]
```diff
@@ -92,7 +93,7 @@
 							<!-- <blocklink onclick='Events.pushevent("show demos", event)'><translate>Recorded demos</translate></blocklink> -->
 						</indent>
 
-						<h2><translate>Options</translate></h2>
+						<h2><translate>Client Options</translate></h2>
```
Comment (necessarily-equal) [inline ui/menu_ingame.rml:96]: Not sure about this specific change

Comment (DolceTriade) [reply ui/menu_ingame.rml:96]: I think it's better to leave it as Options or "Game options"

Comment (bmorel) [reply ui/menu_ingame.rml:96]: Agree, people don't have to know what a client is.

File: ui/menu_ingame.rml [hunk 2]
```diff
@@ -60,8 +60,9 @@
 							<blocklink class="rightfloat" onclick='Events.pushevent("exec disconnect", event)'><translate>Stop server</translate></blocklink>
 							<blocklink class="leftalign" onclick='Events.pushevent("exec pause", event)'><translate>Pause server</translate></blocklink>
 
-							<h2><translate>Map</translate></h2>
+							<h2><translate>Server</translate></h2>
```
Comment (bmorel) [inline ui/menu_ingame.rml:63]: I disagree with this change, as imo saying "server" in a local game might be technically valid, it's still confusing. Not sure if it's better or worst than the previous situation with "Map", though.

**Comment by illwieckz** [comment]
This repository has no longer use as it is not a submodule anymore:

- https://github.com/Unvanquished/Unvanquished/pull/2625

If there is still something to merge from it, this PR should be redone on https://github.com/Unvanquished/Unvanquished instead.

I let this PR open so the one redoing the PR on the other repo can close it when it's redone if it has to be redone, so we know what PR is redone or not yet.

**Pull request closed without merging**